### DISCUSSION TITLE
[Facebook] Always show most recent feed instead of main page

### DISCUSSION THREAD

**Discussion by mbirth** [mshibanami/redirect-web#33] [Redirect Rules]
To not get bombarded with random posts that want to grab your attention, this rule always redirects from the Facebook home page to the "Most Recent" feed where it shows only the posts of your friends sorted from latest to oldest.

* Redirect from: `https://www.facebook.com/` (Mode: **Wildcard**)
* Redirect to: `https://www.facebook.com/?filter=all&sk=h_chr`
* Application: **Default (Safari)**
* Capture Group Processing: \<empty>
* Excluded URL Patterns: \<empty>
* Examples:
  * `https://www.facebook.com` (should show the URL from the *Redirect To* field)
  * `https://www.facebook.com/permalink.php?story_fbid=1234` (should show *No match*)

(Doesn't work inside the Facebook app, of course.)

**Comment by mshibanami** [comment]
Hi @mbirth, this rule works great! Along with [your other rule for Google Maps → Apple Maps](https://github.com/mshibanami/redirect-web/discussions/34#discussioncomment-9745567), could I also add this rule to the Library if you don't mind?

**Reply by mbirth** [reply]
@mshibanami Of course! 😊

**Reply by mshibanami** [reply]
Thanks! I've added it to the library. 🙏

**Comment by karolleon** [comment]
I am trying to use this rule but all I get is that the Facebook page refreshes in an infinite loop.

**Reply by mshibanami** [reply]
Hi @karolleon. Do you use this rule on iPhone?

I investigated.It seems that Facebook’s webpages on iOS are always https://facebook.com…
I’ll update the backend service so this rule won’t appear on the Redirect Web app for iOS. But either way, this rule is probably not functional on your device. Sorry.
